### ISSUE TITLE
opencv-contrib-python-headless VS opencv-contrib-python

### ISSUE THREAD

**Issue by Johndirr** [faustomorales/keras-ocr#48]
Would be nice if keras-ocr would use opencv-contrib-python instead of the headless version. Because the headless version does not have any GUI functionality. This way a user don't has to uninstall and install the desired opencv packages if he wants to have opencv gui functionalities.
What do you think?

**Comment by faustomorales** [comment]
`keras-ocr` doesn't specify OpenCV as a dependency in `setup.cfg` / `setup.py` for the reason you suggest: I also would like users to be able to choose which flavor and version of the OpenCV package they wish to use. The reason that `opencv-python-headless` gets installed is because of the dependency on `imgaug`, for which I've already filed https://github.com/aleju/imgaug/issues/473. It seems the compromise was to look for an existing OpenCV install first but that it doesn't seem to be working.

In other words, I'm not sure there's anything we can do in this package to remedy the matter. Let me know what you think.

**Comment by Johndirr** [comment]
Since we can't do much about it maybe a note in the readme for the used opencv flawor is enough.

**Comment by faustomorales** [comment]
I've added comments describing this issue to 4f7d700bf7cc4619e43ae32f8df47a3293cb43d5. Thanks!
